plaso/parsers/jsonl_plugins/gcp_log.py

```diff
@@ -12,15 +14,27 @@ class GCPLogEventData(events.EventData):
   """Google Cloud (GCP) log event data.
 
   Attributes:
+    caller_ip (str): IP address of the client that requested the operation.
     container (str): TODO
+    dcsa_emails (list[str]): default compute service account attached to a
+        Google Compute Engine (GCE) instance.
+    dcsa_scopes (list[str]): OAuth scopes granted to the default compute service
+        account.
+    delegation_chain (str): service account delegation chain.
     event_subtype (str): JSON event sub type or protocol buffer method.
     event_type (str): TODO
     filename (str): TODO
     firewall_rules (list[str]): firewall rules.
     firewall_source_ranges (list[str]): firewall source ranges.
+    gcloud_command_id (str): unique gcloud command identity.
+    gcloud_command_partial (str): partial gcloud command.
     log_name (str): name of the log entry.
     message (str): TODO
+    method_name (str): operation performed.
+    permissions (list[str]): IAM permission used for the operation.
     policy_deltas (list[str]): TODO
+    principal_email (str): email address of the requester.
```

Comment:
if user is being replaced by principal_email, you would also need to update the corresponding formatter definition

https://github.com/log2timeline/plaso/blob/main/plaso/data/formatters/generic.yaml

Reply:
Fixed.